nss-cohort-37/Bangazon-Site-Green-Monkeys
# Case create product

# Description

User can create a product listing and upload an image.

Fixes #1

## Type of change


# Testing Instructions

Navigate to https://localhost:5001/products
Click 'Create New'
Fill out all fields along with an image
Go to https://localhost:5001/products and check to make sure all information can be viewed.

# Checklist:
